cli/internal/login/link.go

```diff
@@ -57,6 +60,8 @@ func getCmd(config *config.Config, ui cli.Ui) *cobra.Command {
 			link := &link{
 				ui:                  ui,
 				logger:              config.Logger,
+				fsys:                config.Fs,
+				cwd:                 config.Cwd,
```

Comment:
Maybe we change the config to split into two fields and populate them with the same value during the transition?

(These things are not the same, even if we're treating them as the same now.)